libs/ui/tabs/brain/src/lib/brn-tabs-paginated-list.directive.ts

```diff
@@ -160,18 +158,18 @@ export abstract class BrnTabsPaginatedListDirective
 
 	private _injector = inject(Injector);
 
-	constructor(
-		protected _elementRef: ElementRef<HTMLElement>,
-		protected _changeDetectorRef: ChangeDetectorRef,
-		private _viewportRuler: ViewportRuler,
-		@Optional() private _dir: Directionality,
-		private _ngZone: NgZone,
-		private _platform: Platform,
-		@Optional() @Inject(ANIMATION_MODULE_TYPE) public _animationMode?: string,
-	) {
+	protected _elementRef: ElementRef<HTMLElement> = inject(ElementRef);
+	protected _changeDetectorRef: ChangeDetectorRef = inject(ChangeDetectorRef);
```

Comment:
It's not relevant to your actual changes, but these 2 can be marked as private since there's no template to use these in. I'm not sure why it was marked as protected earlier.

Reply:
That makes sense, thanks!